test/OpenCL/Restrictions/access-qualifier.cl

```diff
@@ -0,0 +1,94 @@
+// RUN: %clang_cc1 -verify -pedantic -fsyntax-only -cl-std=CL1.2 %s
+// RUN: %clang_cc1 -verify -pedantic -fsyntax-only -cl-std=CL2.0 %s
+
+typedef image1d_t img1d_ro_default; // expected-note {{previously declared 'read_only' here}}
+
+typedef write_only image1d_t img1d_wo; // expected-note {{previously declared 'write_only' here}}
+typedef read_only image1d_t img1d_ro;
+
+#if __OPENCL_C_VERSION__ >= 200
+typedef read_write image1d_t img1d_rw;
+#endif
+
+#if __OPENCL_C_VERSION__ >= 200
+typedef pipe int pipe_ro; // expected-note {{previously declared 'read_only' here}}
+typedef write_only pipe int pipe_wo; // expected-note {{previously declared 'read_only' here}}
```

Comment:
This note seems to be invalid. It should be "previously declared 'write_only' here".